woocommerce/woocommerce-ios
# Testing: Create UI test target and first test

This gets us started with UI tests with the following changes:

* Adds the `WooCommerceUITests` target with the first test (login/logout with email and password).
* Sets up test plans for unit tests and UI tests.
* Moves resources that are shared between the screenshot test and functional UI tests (Mocks, Screens, Utils) into the WooCommerceUITests target, for clarity in UI test development/maintenance.

To test:

* Confirm `UnitTests` is the default test plan (under the Product > Test Plan menu or in the Xcode test navigator) and all tests pass.
* Run `rake mocks` to start the mock server.
* Select the `UITests` test plan (under the Product > Test Plan menu or in the Xcode test navigator) and run the test in `WooCommerceUITests` to confirm it passes.
* Select the `WooCommerceScreenshots` scheme, run the screenshot test, and confirm it passes.

Update release notes:
